rmw_connext_dynamic_cpp/CMakeLists.txt

```diff
@@ -84,3 +84,6 @@ install(
   LIBRARY DESTINATION lib
   RUNTIME DESTINATION bin
 )
+
+ament_index_get_resource(connext_shared_output_filter "rmw_output_filter" "rmw_connext_shared_cpp")
+ament_index_register_resource("rmw_output_filter" CONTENT "${connext_shared_output_filter}")
```

Comment:
Please add these lines in the same location as for `rmw_connext_cpp` to keep both similar.